src/components/molecules/UserProfilePicture/UserProfilePicture.types.ts

```diff
@@ -1,13 +1,15 @@
 import { User } from "types/User";
 import { WithId } from "utils/id";
 
+// @debt follow our standard patterns and move this back into ./UserProfilePicture.tsx
```

Comment:
Is it possible to move it in this PR, since it's basically a one-minute thing?

Reply:
I didn’t want to as I assumed that would touch a bunch of files that could import this. Didn’t actually check if there were other files that did. Was trying to stick to the ‘focussed PR’ thing (aka not scope creeping), while also pointing out issues as I came across them.

Reply:
IDE says that it's only used in the 1 place, so yeah, happy to move it in this PR:

![image](https://user-images.githubusercontent.com/753891/109251553-cd71eb80-783f-11eb-81c2-285ec208debd.png)

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1215/commits/e65dcb96777870bc51317ace331c43f2c5ce4b0f